lib/xml-parser.js

```diff
@@ -24,12 +24,16 @@ function parse(request, options) {
         options.length = len
     }
     return raw(request, options)
-        .then(str => {
-            return convertXml2Json(str, options.xmlOptions).catch(err => {
+        .then(async (str) => {
+            const xml = await convertXml2Json(str, options.xmlOptions).catch(err => {
```

Comment:
`async/await` is not welcome as the lib should work directly on node v4+ now.

Maybe I will use modern JS next big version.

Reply:
I can change it to use classic `.then` if you're interested in merging the PR

Reply:
It's great if you can update it.

Reply:
I changed it and updated PR